engine-src/Game/LambdaHack/Server/Commandline.hs

```diff
@@ -291,10 +293,16 @@ maxFpsP = optional $ max 1 <$>
               <> help "Display at most N frames per second" )
 
 logPriorityP :: Parser (Maybe Int)
-logPriorityP = optional $ max 0 <$>
-  option auto (  long "logPriority"
-              <> metavar "N"
-              <> help "Log only messages of priority at least N, where 1 (all) is the lowest and 5 (errors only) is the default." )
+logPriorityP = optional $
+  option (auto >>= verifyLogPriority) $
+       long "logPriority"
+    <> metavar "N"
+    <> help "Log only messages of priority at least N, where 1 (all) is the lowest and 5 (errors only) is the default."
+  where
+    verifyLogPriority n = do
+      if (n >= 1 && n <= 5)
```

Comment:
Uh, embarrassingly, I use a hacky priority 0 at

 https://github.com/LambdaHack/LambdaHack/blob/v0.9.5.0/test/test.hs#L17

and I fail to cover that in the documentation of the option. Could you include fix the option doc and also allow the hacky 0 value? Thank you.

Reply:
Oh, I entirely forgot to run tests before making a push :-)